src/WebApi/Infrastructure/Repositories/QuestionListRepository.cs

```diff
@@ -63,7 +63,7 @@ public async Task<bool> RemoveQuestionsFromList(QuestionSetModel questionListMod
             DbContext.Entry(questionList).State = EntityState.Detached;
             return true;
         }
-        catch (DbUpdateException)
+        catch (Exception)
```

Comment:
I am not sure about this. It means any error will be swallowed and no information will be provided. 

I think DbUpdateException may have been the correct option here, but I am wondering what lead you to this change? 🤔

Reply:
When trying to do an invalid update of related entities (eg. adding a non existing question to a question set) the exception thrown by `SaveChangesAsync` is `InvalidOperationException` and not `DbUpdateException`
This could also work
```suggestion
        catch (DbUpdateException)
        {
            return false;
        }
        catch (InvalidOperationException)
        {
            return false;
        }
```

Reply:
This solution could work,  nice work 🚀